### PR TITLE
i2c and spi code bugs

### PR DESCRIPTION
**Pull request by ghost**
'I2cDevice' does not contain a definition for 'GetDefault', same is true for Spi

### CODE REVIEW

**Comment by ivorb** [comment]
@stephenhuang @alecont Will you guys verify for me?

File: en-US/Docs/PinMappingsRPi.md [hunk 1]
```diff
@@ -203,8 +203,10 @@ public async void I2C()
     // 0x40 is the I2C device address
     var settings = new I2cConnectionSettings(0x40);
 
-    // Create an I2cDevice with the default controller and specified I2C settings
-    using (I2cDevice device = await I2cDevice.GetDefault(settings))
```
Comment (bfjelds) [inline en-US/Docs/PinMappingsRPi.md:207]: Yes, this looks like an error.  The method we are encouraging can be found [here](https://github.com/ms-iot/samples/blob/23d8e2d8fbc9d1819c02527cc24582f769082e26/WeatherStation/CS/WeatherStation/StartupTask.cs):

```
            var controller = await I2cController.GetDefaultAsync();
            sensor = controller.GetDevice(new I2cConnectionSettings(0x40));
```

Comment (IoTGirl) [reply en-US/Docs/PinMappingsRPi.md:207]: Thanks @bfjelds !

File: en-US/Docs/PinMappingsRPi.md [hunk 2]
```diff
@@ -253,8 +255,10 @@ public async void SPI()
     // Use chip select line CS0
     var settings = new SpiConnectionSettings(0);
 
-    // Create an SpiDevice with our bus controller and SPI settings
-    using (SpiDevice device = await SpiDevice.GetDefault(settings))
+    // Create an SpiDevice with the specified Spi settings
```
Comment (bfjelds) [inline en-US/Docs/PinMappingsRPi.md:258]: From [https://github.com/ms-iot/samples/blob/23d8e2d8fbc9d1819c02527cc24582f769082e26/RgbLedStrip/CS/apa102/DotStarSample/DotStarStrip.cs](https://github.com/ms-iot/samples/blob/23d8e2d8fbc9d1819c02527cc24582f769082e26/RgbLedStrip/CS/apa102/DotStarSample/DotStarStrip.cs):

```
            var controller = await SpiController.GetDefaultAsync();
            var spiDevice = controller.GetDevice(new SpiConnectionSettings(0)); 
```

**Comment by ivorb** [comment]
@cyberh0me Will you update to the recommended code snippets?

**Pull request merged**